README.md

```diff
@@ -9,22 +9,7 @@ Use SOOS to scan your software for [vulnerabilities](https://app.soos.io/researc
 If you maintain an Open Source project, sign up for the Free as in Beer [SOOS Community Edition](https://soos.io/products/community-edition).
 
 ## Supported Languages and Package Managers
-
-* [C++ - Conan](https://conan.io/center/)
-* [Cargo - Rust](https://doc.rust-lang.org/cargo/)
-* [Composer - PHP](https://maven.apache.org/)
-* [Dart PM (Pub Package Manager) - Dart](https://pub.dev/)
-* [Go Modules - Go (GoLang)](https://pkg.go.dev/)
-* [Gradle - Java & Kotlin](https://gradle.org/)
-* [Homebrew - (various languages)](https://brew.sh/)
-* [Maven - Java](https://maven.apache.org/)
-* [Mix - Elixir](https://hexdocs.pm/mix/Mix.html)
-* [NuGet - .NET](https://www.nuget.org/)
-* [NPM (Node Package Manager) - Node](https://www.npmjs.com/)
-* [PyPI - Python](https://pypi.org/)
-* [Rebar3 - Erlang](https://rebar3.readme.io/docs/getting-started)
-* [Ruby Gems - Ruby](https://rubygems.org/)
-* And more!
+Our full list of supported manifest formats can be found [here](https://kb.soos.io/help/soos-languages-supported).
```

Comment:
No functional changes needed, but changed documentation to match SCA script and Azure task.